app/assets/stylesheets/components/_header.sass

```diff
@@ -1,12 +1,21 @@
 $Header-color-background: Theme-color(background, menu-bar)
 $Header-height-header: 70px
 $Header-spacing-base: $Theme-spacing-base
-$Header-logo-width: 200px
+$Header-logo-width: 230px
+$Header-navbar: Theme-fontSize(menu-bar, font-size)
 
 .Header
   background-color: $Header-color-background
-  height: $Header-height-header
-  text-align: center
+  font-size: 0
+  text-align: right
```

Comment:
Are you talking about `text-align: right` ? if i dont do that, the descendants stick together aligned to the left.